README.md

```diff
@@ -24,85 +32,162 @@ See the [Quickstart](https://warpforge.github.io/warpgrapher/warpgrapher/quickst
 
 See the [Warpgrapher Book](https://warpforge.github.io/warpgrapher/) for in-depth usage documentation. 
 
-## Contributing
+# Contributing
 
 Note that the steps below are for doing development on the Warpgrapher itself,
 to contribute to the project. In order to develop with Warpgrapher, on your own
 project, see the Documentation.
 
-### Clone the Warpgrapher Repository
+## Clone the Warpgrapher Repository
 
 ```
 git clone https://github.com/warpforge/warpgrapher.git
 ```
 
-### Build Warpgrapher
+## Build Warpgrapher
+
+To build for use with Cosmos DB:
 
 ```bash
-cargo build
+cargo build --features cosmos
 ```
 
-### Test
+To build for use with Neo4J:
 
-Set env variables:
+```bash
+cargo build --features neo4j
+```
+
+## Test
+
+### Set Environment Variables
+
+For Cosmos DB graphs:
 
 ```bash
-export DB_PASS=my-db-pass
-export DB_URL=http://neo4j:${DB_PASS}@127.0.0.1:7474/db/data
+export WG_COSMOS_HOST=*MY-COSMOS-DB*.gremlin.cosmos.azure.com
+export WG_COSMOS_PORT=443
+export WG_COSMOS_USER=/dbs/*MY-COSMOS-DB*/colls/*MY-COSMOS-COLLECTION*
+export WG_COSMOS_PASS=*MY-COSMOS-KEY*
 ```
 
-Run neo4j database:
+For Neo4J:
 
 ```bash
-docker run --rm -e NEO4J_AUTH="neo4j/${DB_PASS}" -p 7474:7474 -p 7687:7687 neo4j:3.5
+export WG_NEO4J_HOST=127.0.0.1
+export WG_NEO4J_PORT=7687
+export WG_NEO4J_USER=neo4j
+export WG_NEO4J_PASS=*MY-DB-PASS*
 ```
 
-Run unit tests:
+### Run the Database
+
+For Cosmos DB:
+
+Cosmos DB is an Azure cloud service, so it's already running. Or, if you're using a local Cosmos
+emulator, start the service based on its instructions.
+
```

Comment:
Let's add a note here about the partition key.

Reply:
Agreed.  Added.